src/containers/Bounty/components/modals/ActivateDeadFormModal.js

```diff
@@ -58,7 +75,13 @@ const ActivateDeadFormModal = props => {
           >
             Cancel
           </Button>
-          <Button type="action">Re-Activate</Button>
+          <Button
+            type="action"
+            disabled={submitFailed && invalid}
+            loading={asyncValidating && typeof asyncValidating === 'boolean'}
```

Comment:
only show the button's loader if async validation is occurring AND the field that initiated the async validation is the submit button (denoted by asyncValidating being `true` vs. a field name like `balance`).